src/app/app.component.scss

```diff
@@ -0,0 +1 @@
+@import '../stylesheets/variables';
```

Comment:
@carryall Do we need to include it here? as we included `@import './stylesheets/application';` in the global style `style.scss` then it should in include/apply all the  global css(style.scss) to any component css. What do you think?